### DISCUSSION TITLE
Button color on hover

### DISCUSSION THREAD

**Discussion by tecticsolutions** [add2cal/add-to-calendar-button#373] [Q&A] [answered]
Hi,

I'm using this shortcode in wordpress with some style at last parameter, but I don't know why, the **--btn-text-hover:#000;"** it's not working, 

[add-to-calendar-button name="NHMRC Call for Applications: Genomics Health Futures Mission Grant Opportunity 2022" dates='{"name":"Grant - Minimum due data Genomics Health Futures Mission Grant Opportunity 2022, NHMRC","startDate":"2023-02-01"},{"name":"Grant - Deadline Genomics Health Futures Mission Grant Opportunity 2022, NHMRC","startDate":"2023-02-22"}' options="'Apple','Google','iCal','Microsoft365','Outlook.com'" hideCheckmark="true" hideBackground="true" buttonStyle="round" styleLight="--font:Roboto;--btn-background: #F04E6E;--btn-text:#fff;--btn-text-hover:#000;" ]

[screen-recording.webm](https://user-images.githubusercontent.com/128813647/227526644-b4ff4c2a-73ec-4083-b3ad-7604f597ae62.webm)

I've tried adding an !important with no result.

Is strange because if I use the inspector i see the overrides correctly

![bkg](https://user-images.githubusercontent.com/128813647/227526696-b9efec79-5b91-4c23-b146-f789575e05f6.jpg)

Any help will be apreciated

**Comment by jekuer** [comment]
Unfortunately, this does not work at the moment.
I would not say it's a "bug", but definitely not ideal.
It has been already optimized on a working branch and will be published with the next minor release very soon.

Until then, I would recommend to use a lighter `--btn-background` and a black font for the moment.

**Comment by jekuer** [comment, marked as answer]
@tecticsolutions , should work now with latest version 2.2.5 😃

**Reply by tecticsolutions** [reply]
Like a charm! thanks!
